lib/src/main/java/graphql/nadel/engine/transform/hydration/NadelHydrationTransform.kt

```diff
@@ -201,17 +201,28 @@ internal class NadelHydrationTransform(
             ).map { actorQuery ->
                 async {
                     val hydrationSourceService = executionBlueprint.getServiceOwning(instruction.location)!!
-                    engine.executeTopLevelField(
-                        service = instruction.actorService,
-                        topLevelField = actorQuery,
-                        executionContext = executionContext,
-                        serviceHydrationDetails = ServiceExecutionHydrationDetails(
-                            instruction.timeout,
-                            1,
-                            hydrationSourceService,
-                            instruction.location
-                        )
+                    val serviceHydrationDetails = ServiceExecutionHydrationDetails(
+                        instruction.timeout,
+                        1,
+                        hydrationSourceService,
+                        instruction.location
                     )
+                    if (executionContext.hints.removeHydrationSpecificExecutionCode) {
```

Comment:
I am confised by this feature flag hint - is it reversed?

Reply:
I think false is current behavior, true is new behavior (i.e. true is as if code was removed).